community/mtd-utils/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=mtd-utils
+pkgver=2.1.0
+pkgrel=0
+pkgdesc="Utilities for handling MTD devices, and for dealing with FTL, NFTL JFFS2, etc."
+url="http://www.linux-mtd.infradead.org/"
+arch="all"
+license="GPL-2.0"
+makedepends="
+	acl-dev
+	linux-headers
+	lzo-dev
+	openssl-dev
+	util-linux-dev
+	zlib-dev
+"
+subpackages="${pkgname}-doc"
+source="ftp://ftp.infradead.org/pub/mtd-utils/${pkgname}-${pkgver}.tar.bz2"
+
+build()
+{
+	./configure \
+	            --build="${CBUILD}" \
+	            --host="${CHOST}" \
+	            --prefix="/usr" \
+	            --sysconfdir="/etc" \
+	            --mandir="/usr/share/man" \
+		|| return 1
+	make \
+		|| return 1
+}
+
+package()
+{
+	make install -j1 \
```

Comment:
```suggestion
	make -j1 DESTDIR="$pkgdir" install -j1
```

Reply:
Was the duplicated `-j1` intended?

Reply:
copy paste thing whilst moving the install from in between to the end I suspect

Reply:
Was a mistake, both cases can properly be removed.